sentry/src/event_reducer.rs

```diff
@@ -104,15 +102,8 @@ mod test {
             referrer: None,
         };
 
-        let session = Session {
-            ip: Default::default(),
-            country: None,
-            referrer_header: None,
-            os: None,
-        };
-
         for i in 0..101 {
-            reduce(&logger, &channel, &mut event_aggr, &event, &session)
+            reduce(&channel, &mut event_aggr, &event, &None)
```

Comment:
Uhhh... ohhh... this should be fixed. The payout here should be 1.
Bellow where the `even_payouts` is fetched and `assert!`ed it actually gets the `impression_event.event_counts` as well, which is a typo!